src/ctelnet.cpp

```diff
@@ -480,11 +480,12 @@ void cTelnet::slot_socketDisconnected()
     QTime timeDiff(0, 0, 0, 0);
     msg = tr("[ INFO ]  - Connection time: %1\n    ")
                   .arg(timeDiff.addMSecs(mConnectionTimer.elapsed())
-                               .toString(tr("hh:mm:ss.zzz",
-                                            // Intentional comment to separate arguments
-                                            "This is the format to be used to show the profile connection time, it follows "
-                                            "the rules of the \"QDateTime::toString(...)\" function and may need "
-                                            "modification for some locales, e.g. France, Spain.")));
+                                /*:
+                                This is the format to be used to show the profile connection time, it follows
+                                the rules of the \"QDateTime::toString(...)\" function and may need
+                                modification for some locales, e.g. France, Spain.
+                                */
+                               .toString(tr("hh:mm:ss.zzz")));
```

Comment:
Since this is no longer a string argument to a function the double-quotes do not need escaping now:
```suggestion
                                /*:
                                This is the format to be used to show the profile connection time, it follows
                                the rules of the "QDateTime::toString(...)" function and may need
                                modification for some locales, e.g. France, Spain.
                                */
                               .toString(tr("hh:mm:ss.zzz")));
```

Reply:
Yeah, I got this right in many places, but I missed this somehow.